File: go/usage_plan_fee.go
// this file is @generated
package meteroid

// Usage-based fee
type UsagePlanFee struct {
	Cadence BillingPeriodEnum `json:"cadence"`

	MetricId BillableMetricId `json:"metric_id"`

	Pricing PlanUsagePricingModel `json:"pricing"`
}

File: python/meteroid/models/usage_plan_fee.py
# this file is @generated
import dataclasses

from ..serialization import BaseModel
from .billable_metric_id import BillableMetricId
from .billing_period_enum import BillingPeriodEnum
from .plan_usage_pricing_model import PlanUsagePricingModel


@dataclasses.dataclass
class UsagePlanFee(BaseModel):
    """Usage-based fee"""

    cadence: BillingPeriodEnum

    metric_id: BillableMetricId

    pricing: PlanUsagePricingModel
